src/bin/openst.js

```diff
@@ -10,17 +10,55 @@ const { version } = require('../../package.json');
 
 program
   .version(version)
-  .name('bt')
-  .arguments('openst <config> <organization> <eip20Token>')
+  .name('openst')
+  .description('An executable to setup OpenST.');
+
+program.command('openst <config> <eip20Token> ')
   .description('An executable to setup OpenST.')
   .action(
-    async (config, organization, eip20Token) => {
+    async (config, eip20Token) => {
       await connected.run(
         config,
         async (chainConfig, connection) => {
-          const openst = new OpenST(chainConfig, connection);
-          await openst.setupOpenst(organization, eip20Token);
-          chainConfig.write(config);
+          try {
+            const openst = new OpenST(chainConfig, connection);
+            await openst.setupOpenst(eip20Token);
+            chainConfig.write(config);
+          } catch (e) {
+            console.error(e);
```

Comment:
Use logger here.
Also on exception we should do Promise.reject.

Reply:
When logger is used, error are reported incorrectly.
It reports the error as `error: undefined`. This doesn't help in debugging.

Reply:
Below should work:
```
logger.error(`message: ${e.message}, backtrace: ${e.backtrace}`)
```